packages/gatsby-link/src/index.tsx

```diff
@@ -62,23 +91,24 @@ class GatsbyLink extends React.Component {
       IOSupported,
     }
     this.handleRef = this.handleRef.bind(this)
+    this.defaultGetProps = this.defaultGetProps.bind(this)
```

Comment:
tbh, i'd rather run this the other way. Let's change the handleRef and defaultGetProps to static arrow functions

Reply:
Do you mean something like:

```tsx
  static defaultGetProps = <TState,>({ isPartiallyCurrent, isCurrent }: LinkGetProps, props: IGatsbyLinkProps<TState>): IGetProps => {
    if (props.partiallyActive ? isPartiallyCurrent : isCurrent) {
      return {
        className: [props.className, props.activeClassName]
          .filter(Boolean)
          .join(` `),
        style: { ...props.style, ...props.activeStyle },
      }
    }
    return {}
  }
```

Reply:
almost. not quite that intense! haha

It just means this:
```js
class Foo {
  // this approach has the `this` bound to the class 
  defaultGetProps = () => {}

  // this approach does not, and requires binding in the constructor
  defaultGetProps() {}
}
```

No need for the `static`

Reply:
Of course 🤦‍♂ that's done now